lib/mdx.mli

```diff
@@ -69,6 +69,16 @@ val parse_lexbuf: syntax -> Lexing.lexbuf -> t
 
 (** {2 Evaluation} *)
 
+val run_to_stdout : ?syntax: syntax -> f:(string -> t -> string) -> string -> unit
+
+val run_to_file :
+  ?syntax: syntax ->
+  ?force_output: bool ->
+  f:(string -> t -> string) ->
+  outfile: string ->
+  string ->
+  unit
```

Comment:
Can you add a little documentation ?
I see no problem in moving this function below `run` so it can reference it.

Reply:
I did it the other way around, I documented the `run_to_file` and `run_to_stdout` functions and describe `run` in term of `run_to_file`